pkg/resourcecollector/persistentvolume.go

```diff
@@ -16,28 +16,22 @@ func (r *ResourceCollector) pvToBeCollected(
 	object runtime.Unstructured,
 	namespace string,
 ) (bool, error) {
-	phase, err := collections.GetString(object.UnstructuredContent(), "status.phase")
-	if err != nil {
-		return false, err
+	var pv v1.PersistentVolume
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(object.UnstructuredContent(), &pv); err != nil {
+		return false, fmt.Errorf("error converting to persistent volume: %v", err)
 	}
+
 	// Only collect Bound PVs
-	if phase != string(v1.ClaimBound) {
+	if pv.Status.Phase != v1.VolumeBound {
 		return false, nil
 	}
 
+	pvcName := pv.Spec.ClaimRef.Name
```

Comment:
I guess,  need to add nil check for `ClaimRef`

Reply:
Fixed